pkg/cmd/cli/secrets/add_secret_to_obj.go

```diff
@@ -32,6 +32,11 @@ To use your secret inside of a pod or as a push, pull, or source secret for a bu
 To use your secret as an image pull secret, you must add a 'pull' secret to your service account like this:
 
   $ %s serviceaccount/sa-name secrets/secret-name --for=pull
+
+To both mount your secret and use it for image pulls, you can specify both roles for the secret with either of the following:
+
+  $ %s serviceaccount/sa-name secrets/secret-name --for=pull --for=mount
+  $ %s serviceaccount/sa-name secrets/secret-name --for=pull,mount
```

Comment:
@liggitt @smarterclayton  this will invariably be a point of discussion - do we need to expose both ways to do this? Is it cleaner to just expose one? If so, I vote the second one